### ISSUE TITLE
Two debug tabs is opened in IDEA 14

### ISSUE THREAD

**Issue by viatra2** [viatra2/IDEA-AWS-Elastic-Beanstalk-Integration#26]
Two debug tabs is opened for one debug session when starting remote debugging in IDEA 14.
Debugger is operational.
![debug_views](https://cloud.githubusercontent.com/assets/4850347/5192249/8b33cfcc-74fe-11e4-9fa9-483c1f35ded9.PNG)

**Comment by viatra2** [comment]
"_Throwable: startNotify called already_" error appears in Event log when tabs are created.

```
java.lang.Throwable
    at com.intellij.openapi.diagnostic.Logger.error(Logger.java:115)
    at com.intellij.execution.process.ProcessHandler.startNotify(ProcessHandler.java:70)
    at com.intellij.debugger.engine.RemoteDebugProcessHandler.startNotify(RemoteDebugProcessHandler.java:42)
    at com.intellij.remoteServer.impl.runtime.deployment.debug.JavaDebuggerLauncherImpl.startDebugSession(JavaDebuggerLauncherImpl.java:54)
    at com.intellij.remoteServer.impl.runtime.deployment.debug.JavaDebuggerLauncherImpl.startDebugSession(JavaDebuggerLauncherImpl.java:36)
    at com.intellij.remoteServer.impl.runtime.ServerConnectionImpl$DeploymentOperationCallbackImpl$1.run(ServerConnectionImpl.java:377)
    at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:314)
    at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:209)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:672)
    at java.awt.EventQueue.access$400(EventQueue.java:81)
    at java.awt.EventQueue$2.run(EventQueue.java:633)
    at java.awt.EventQueue$2.run(EventQueue.java:631)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:642)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:746)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:576)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:383)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
```

"_IllegalArgumentException: Argument for @NotNull parameter 'executionConsole' of com/intellij/xdebugger/impl/XDebuggerManagerImpl.getDebugSession must not be null_" error message appears on closing debugger tab

```
java.lang.IllegalArgumentException: Argument for @NotNull parameter 'executionConsole' of com/intellij/xdebugger/impl/XDebuggerManagerImpl.getDebugSession must not be null
    at com.intellij.xdebugger.impl.XDebuggerManagerImpl.getDebugSession(XDebuggerManagerImpl.java)
    at com.intellij.debugger.ui.DebuggerPanelsManager.getSession(DebuggerPanelsManager.java:226)
    at com.intellij.debugger.ui.DebuggerPanelsManager.access$100(DebuggerPanelsManager.java:53)
    at com.intellij.debugger.ui.DebuggerPanelsManager$2.contentSelected(DebuggerPanelsManager.java:161)
    at sun.reflect.GeneratedMethodAccessor600.invoke(Unknown Source)
    at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:25)
    at java.lang.reflect.Method.invoke(Method.java:597)
    at com.intellij.util.messages.impl.MessageBusConnectionImpl.deliverMessage(MessageBusConnectionImpl.java:115)
    at com.intellij.util.messages.impl.MessageBusImpl.doPumpMessages(MessageBusImpl.java:315)
    at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:302)
    at com.intellij.util.messages.impl.MessageBusImpl.pumpMessages(MessageBusImpl.java:294)
    at com.intellij.util.messages.impl.MessageBusImpl.sendMessage(MessageBusImpl.java:287)
    at com.intellij.util.messages.impl.MessageBusImpl.access$200(MessageBusImpl.java:41)
    at com.intellij.util.messages.impl.MessageBusImpl$2.invoke(MessageBusImpl.java:177)
    at com.sun.proxy.$Proxy148.contentSelected(Unknown Source)
    at com.intellij.execution.ui.RunContentManagerImpl$4.selectionChanged(RunContentManagerImpl.java:159)
    at com.intellij.ui.content.impl.ContentManagerImpl.fireSelectionChanged(ContentManagerImpl.java:621)
    at com.intellij.ui.content.impl.ContentManagerImpl.addSelectedContent(ContentManagerImpl.java:426)
    at com.intellij.ui.content.impl.ContentManagerImpl$2.run(ContentManagerImpl.java:515)
    at com.intellij.ui.content.impl.ContentManagerImpl.setSelectedContent(ContentManagerImpl.java:538)
    at com.intellij.ui.content.impl.ContentManagerImpl.setSelectedContentCB(ContentManagerImpl.java:484)
    at com.intellij.ui.content.impl.ContentManagerImpl.setSelectedContentCB(ContentManagerImpl.java:473)
    at com.intellij.ui.content.impl.ContentManagerImpl.setSelectedContentCB(ContentManagerImpl.java:557)
    at com.intellij.ui.content.impl.ContentManagerImpl.removeContent(ContentManagerImpl.java:302)
    at com.intellij.ui.content.impl.ContentManagerImpl.removeContent(ContentManagerImpl.java:218)
    at com.intellij.ui.content.tabs.TabbedContentAction$CloseAction.actionPerformed(TabbedContentAction.java:74)
    at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAware(ActionUtil.java:164)
    at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter$1.run(ActionMenuItem.java:312)
    at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:926)
    at com.intellij.openapi.wm.impl.IdeFocusManagerImpl.runOnOwnContext(IdeFocusManagerImpl.java:124)
    at com.intellij.openapi.actionSystem.impl.ActionMenuItem$ActionTransmitter.actionPerformed(ActionMenuItem.java:282)
    at javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1995)
    at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.java:109)
    at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:512)
    at com.intellij.ui.plaf.beg.BegMenuItemUI.access$300(BegMenuItemUI.java:44)
    at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:532)
    at java.awt.Component.processMouseEvent(Component.java:6297)
    at javax.swing.JComponent.processMouseEvent(JComponent.java:3275)
    at java.awt.Component.processEvent(Component.java:6062)
    at java.awt.Container.processEvent(Container.java:2039)
    at java.awt.Component.dispatchEventImpl(Component.java:4660)
    at java.awt.Container.dispatchEventImpl(Container.java:2097)
    at java.awt.Component.dispatchEvent(Component.java:4488)
    at java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4575)
    at java.awt.LightweightDispatcher.processMouseEvent(Container.java:4236)
    at java.awt.LightweightDispatcher.dispatchEvent(Container.java:4166)
    at java.awt.Container.dispatchEventImpl(Container.java:2083)
    at java.awt.Window.dispatchEventImpl(Window.java:2489)
    at java.awt.Component.dispatchEvent(Component.java:4488)
    at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:674)
    at java.awt.EventQueue.access$400(EventQueue.java:81)
    at java.awt.EventQueue$2.run(EventQueue.java:633)
    at java.awt.EventQueue$2.run(EventQueue.java:631)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:98)
    at java.awt.EventQueue$3.run(EventQueue.java:647)
    at java.awt.EventQueue$3.run(EventQueue.java:645)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.security.AccessControlContext$1.doIntersectionPrivilege(AccessControlContext.java:87)
    at java.awt.EventQueue.dispatchEvent(EventQueue.java:644)
    at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:746)
    at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:572)
    at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:383)
    at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:269)
    at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:184)
    at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:174)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:169)
    at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:161)
    at java.awt.EventDispatchThread.run(EventDispatchThread.java:122)
```

**Comment by viatra2** [comment]
This issue is caused by code external to our plugin. See https://youtrack.jetbrains.com/issue/IDEA-133575 for details

**Comment by viatra2** [comment]
Fixed in platform of IDEA 14.0.2
